src/Microsoft.SqlTools.ServiceLayer/Connection/ReliableConnection/ReliableConnectionHelper.cs

```diff
@@ -26,6 +27,8 @@ internal static class ReliableConnectionHelper
         private const string ApplicationIntent = "ApplicationIntent";
         private const string MultiSubnetFailover = "MultiSubnetFailover";
         private const string DacFxApplicationName = "DacFx";
+
+        private const int SqlDwEngineEditionId = (int)DatabaseEngineEdition.SqlDataWarehouse;
```

Comment:
The engine version it is compared to is an integer, so I can either cast it here or later on when I compare it. I decided to do it here because the DacFx code had an explicit '= 6' instead of a reference the enum. So if we ever port over any other code that uses this variable, things will be consistent.